Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -22,7 +22,7 @@ public class BoxPlotSeries : XYAxisSeries
         /// <summary>
         /// The default tracker format string
         /// </summary>
-        public new const string DefaultTrackerFormatString = "{0}\n{1}: {2}\nUpper Whisker: {3:0.00}\nThird Quartil: {4:0.00}\nMedian: {5:0.00}\nFirst Quartil: {6:0.00}\nLower Whisker: {7:0.00}";
+        public new const string DefaultTrackerFormatString = "{0}\n{1}: {2}\nUpper Whisker: {3:N2}\nThird Quartil: {4:N2}\nMedian: {5:N2}\nMean: {6:N2}\nFirst Quartil: {7:N2}\nLower Whisker: {8:N2}";
```

Comment:
Please add it to the changelog, All custom TrackFormatStrings will have to be updated.